tools/automation.py

```diff
@@ -288,37 +288,46 @@ def get_new_mentors_in_yml_format(yml_file_path, xlsx_file_path, skiprows=1):
                 new_index += 1
                 mentors.append(mentor)
 
-        print(f"Added {len(mentors)} mentors to the mentors.yml file")
+        logging.info(f"Added {len(mentors)} mentors to the mentors.yml file")
     else:
-        mentors = get_all_mentors_in_yml_format(xlsx_file_path, skiprows)
+        mentors = get_all_mentors_in_yml_format(xlsx_file_path, skip_rows)
     return mentors
 
 
-def main():
+def run_automation():
     logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
-    if len(sys.argv) != 4:
-        logging.error("Usage: python SCRIPT_NAME FILE_PATH_MENTORS_XLSX FILE_PATH_MENTORS_YML MODE")
-        sys.exit(1)
-
-    xlsx_file_path = sys.argv[1]
-    yml_file_path = sys.argv[2]
-    mode = sys.argv[3]
-    # TODO: skiprows = sys.argv[4]
-    skiprows = 1
-
-    # Append new mentors to the current mentors.yml file
-    if mode == WriteMode.APPEND.value:
-        list_of_mentors = get_new_mentors_in_yml_format(yml_file_path, xlsx_file_path, skiprows)
+
+    if len(sys.argv) == 5:
+        xlsx_file_path = sys.argv[1]
+        yml_file_path = sys.argv[2]
+        mode = WriteMode(sys.argv[3])
+        skip_rows = sys.argv[4]
+
+        logging.info("Params: xlsx: %s yml: %s mode: %s skip_rows: %s", xlsx_file_path, yml_file_path, mode, skip_rows)
+    else:
+        xlsx_file_path = "samples/mentors.xlsx"
+        yml_file_path = "samples/mentors.yml"
+        mode = WriteMode.APPEND
+        skip_rows = 0
+
+        logging.info("Default values: xlsx: %s yml:: %s mode: %s", xlsx_file_path, yml_file_path, mode)
+
+    if mode == WriteMode.APPEND:
+        logging.info("Appending option selected.")
+
+        list_of_mentors = get_new_mentors_in_yml_format(yml_file_path, xlsx_file_path, skip_rows=skip_rows)
+
+        logging.info("New Mentors size: %d", len(list_of_mentors))
```

Comment:
Maybe this info is not needed here since the info about added mentors is displayed just above this one:
```
2024-05-06 17:49:15,366 - INFO - Added 1 mentors to the mentors.yml file
2024-05-06 17:49:15,366 - INFO - New Mentors size: 1
```

Unless you have wanted to display the new mentors.yml size (old mentors + added mentors)?